src/execution/vm/llvm_engine.cpp

```diff
@@ -948,29 +951,32 @@ void LLVMEngine::CompiledModuleBuilder::Simplify() {
 }
 
 void LLVMEngine::CompiledModuleBuilder::Optimize() {
-  llvm::legacy::PassManager module_passes;
   llvm::legacy::FunctionPassManager function_passes(llvm_module_.get());
 
-  // Add the appropriate TargetLibraryInfo and TargetTransformInfo
-  auto target_library_info_impl = std::make_unique<llvm::TargetLibraryInfoImpl>(target_machine_->getTargetTriple());
+  // Add the appropriate TargetLibraryInfo and TargetTransformInfo.
+  auto tli = std::make_unique<llvm::TargetLibraryInfoImpl>(target_machine_->getTargetTriple());
```

Comment:
@pmenon Is this unique pointer necessary? It doesn't seem to be used. I'm not sure whether it has side effects that we want.

Reply:
No, it isn't necessary. Good catch!